Cargo.toml

```diff
@@ -40,6 +40,9 @@ env_logger = { version = "0.3.4", optional = true }
 lazy_static = { version = "0.2.0", optional = true }
 rustyline = { version = "1.0.0", optional = true }
 
+# Binding crates
+regex = "0.2.1"
```

Comment:
Could you make this optional and then add `regex` as a default feature. That way it is possible to avoid the dependency if necessary.